src/resource/user/user.service.ts

```diff
@@ -1,6 +1,8 @@
 import { Injectable, Logger } from '@nestjs/common';
-import { CreateUserInput } from './dto/create-user.input';
+import { isNil } from 'lodash';
 import 'crypto';
+
+import { CreateUserInput } from './dto/create-user.input';
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const addHours = require('date-fns/addHours');
```

Comment:
Why do we need to disable eslint here?

Reply:
Maybe I didn't configure eslint correctly? It doesn't like the use of `require` here, and somehow for this package `date-fns/addHours` doesn't work with `import`

Reply:
Please take a look at this [https://date-fns.org/v2.28.0/docs/addHours](url), it should work with the correct importing way.

Reply:
Nice! So we were using the incorrect import in the past, `import { addHours } from 'date-fns';` works well.